docker/README.md

```diff
@@ -79,19 +79,19 @@ To pull the a particular container, run the following command
 #### Pulling the latest tag:
 Docker pull by default pulls the latest tag. This tag isn't available until after an official release. 
 ```bash
-docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release which is the same as 1.0.0-mxnet-cpu
+docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release which is the same as 1.0.1-mxnet-cpu
```

Comment:
```suggestion
docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release of mxnet-cpu image
```

Reply:
Lets pull in these changes after 1.0.2 release.